IPIP/0342-content-router-discovery.md

```diff
@@ -32,6 +32,20 @@ is also insufficient long term because:
 
 ## Detailed design
 
+This spec is designed for the ability of IPFS nodes to automatically discover
+and make use of 'content routers'. Content routers are services which are able
+to fulfill libp2p's [ContentRouting](https://github.com/libp2p/go-libp2p/blob/master/core/routing/routing.go#L26)
```

Comment:
nit: this is not libp2p API, it is there due to legacy reasons (libp2p being part of go-ipfs long time ago).
afaik libp2p team removed specs for this from their repo and we have a gap  in IPFS one (see https://github.com/ipfs/specs/issues/345).

```suggestion
to fulfill IPFS [ContentRouting](https://github.com/libp2p/go-libp2p/blob/master/core/routing/routing.go#L26)
```